src/detect-engine-filedata.c

```diff
@@ -99,38 +99,43 @@ static const uint8_t *DetectEngineSMTPGetBufferForTX(uint64_t tx_id,
     *stream_start_offset = 0;
     uint64_t file_size = FileDataSize(curr_file);
 
-    if (det_ctx->smtp_buffers_list_len == 0) {
-        if (SMTPCreateSpace(det_ctx, 1) < 0)
+    const uint32_t content_limit = de_ctx->filedata_config[f->alproto].content_limit;
+    const uint32_t content_inspect_min_size = de_ctx->filedata_config[f->alproto].content_inspect_min_size;
+    // TODO this is unused, is that right?
```

Comment:
Should this be removed ?

Reply:
I prefer leaving it as it is something odd that needs to be checked some day. But it doesn't need to block the merge.